src/msteams-app.css

```diff
@@ -0,0 +1,163 @@
+@charset "UTF-8";
+
+body {
+  font-family:  "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif !important;
+  font-size: 1.4rem;
+  min-width: auto;
+}
+a {
+  color: #5558af;
+}
+a:hover, a:focus {
+  text-decoration: underline;
+  outline: none;
+}
+
+/* Form controls */
+.form-field-title:first-of-type {
+  margin-top: 0;
+}
+.form-field-title {
+  display: flex;
+  margin: 1rem 0 .3rem;
+}
+.form-control:focus {
+  border-bottom-color: #5558af;
+  box-shadow: none;
+  outline: 0;
+}
+.form-control {
+  background: #eef1f5;
+  border: 0 solid transparent;
+  border-bottom-width: .2rem;
+  border-radius: .3rem;
+  color: #16233a;
+  resize: none;
+  box-shadow: none;
+  display: block;
+  width: 100%;
+  height: 34px;
+  padding: 6px 12px;
+  font-family: inherit;
+  font-size: 14px;
+  line-height: 1.4286;
+}
+input, textarea {
+    font-family: inherit;
+}
+
+/* Buttons */
+.ts-btn {
+  background: 0;
+  border: .2rem solid transparent;
+  border-radius: .3rem;
+  font-size: 1.4rem;
+  padding: .4rem 3.2rem;
+  height: 3.2rem;
+  min-width: 3.2rem;
+  white-space: nowrap;
+}
+.ts-btn:hover {
+  outline: 0;
+  text-decoration: none
+}
+.ts-btn-primary {
+  background: #5558af;
+  outline-style: none;
+  color: #fff
+}
+.ts-btn-primary:focus,
+.ts-btn-primary:hover {
+  background: #4c509d;
+  color: #fff;
+  cursor: pointer;
+}
+
+/* Generic section styles */
+.section-caption {
+  font-size: 1.2rem;
+  margin: .5rem 0 .5rem 0;
+  color: rgba(22,35,58,.74);
+}
+
+/* Login area */
+.login-container {
+  margin: 0 0 2.0rem 0;
+}
+.logout-link {
+  margin-left: .6rem;
+}
+
+/* Error list */
+.error-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: -2.0rem;
+}
+.error-title {
+  font-size: 1.8rem;
+  font-weight: 600;
+}
+.error-subtitle {
+  margin: 0 0 1.6rem 0;
+}
+
+/* Settings area */
+.settings-container {
+  margin: 0 0 2.0rem 0;
+}
+
+/* Dark theme overides */
+body.theme-dark {
+  background: #404045;
+  color: #fff;
+}
+.theme-dark a {
+  color: #9fa4fe;
+}
+.theme-dark .form-control {
+  background: #2b2b30;
+  color: #fff;
+}
+.theme-dark .ts-btn-primary {
+  background: #9fa4fe;
+  color: #2b2b30;
+}
+.theme-dark .ts-btn-primary:focus,
+.theme-dark .ts-btn-primary:hover {
+  background: #696cb9;
+  color: #fff;
+  cursor: pointer;
+}
+.theme-dark .section-caption {
+  color: rgba(255,255,255,.74);
+}
+
+/* High-contrast theme overides */
+body.theme-contrast {
+  background: #000;
+  color: #fff;
+}
+.theme-contrast a {
+  color: #ffff01;
+}
+.theme-contrast .form-control {
+  background: #000;
+  color: #fff;
+}
+.theme-contrast .ts-btn-primary {
+  background: #000;
+  border: .2rem solid #fff;
+  color: #fff;
+}
+.theme-contrast .ts-btn-primary:focus,
+.theme-contrast .ts-btn-primary:hover {
+  background: #ffff01;
+  color: #000;
+  cursor: pointer;
+}
+.theme-contrast .section-caption {
+  color: #fff;
+}
+

```

Comment:
1. It was generated manually based on what we did for the Trello tab.
2. It should be versioned just like the SDK. 
3. Yes it would be on our CDN just like our SDK which also answers the versioning question because all the CDN paths are versioned.